apps/coscrad-frontend/src/utils/generic-components/presenters/external-link-presenter.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Link, Tooltip, Typography } from '@mui/material';
+
+interface ExternalLinkProps {
+    url: string;
+}
+
+export const ExternalLinkPresenter = ({ url }: ExternalLinkProps): JSX.Element => {
+    const linkDisplayLength = 30;
+
+    const displayURL =
+        url.length > linkDisplayLength ? `${url.substring(0, linkDisplayLength)}...` : url;
+
+    return (
+        <Box mb={1}>
+            <Typography component={'span'} sx={{ fontWeight: 'bold' }}>
+                External Link:
+            </Typography>
+            {` `}
```

Comment:
I guess this is a standard approach, but man does it look funny.

Reply:
see comment above.